src/components/get-members.tsx

```diff
@@ -0,0 +1,34 @@
+import { getMembers } from "@/lib/firebase_utils";
```

Comment:
Is this import now correct? When you changed the filename to `firebase.utils.ts`?